src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -262,7 +266,44 @@ private void initDragAndDrop() {
                             for (Tab libraryTab : tabbedPane.getTabs()) {
                                 if (libraryTab.getId().equals(tab.getId()) &&
                                         !tabbedPane.getSelectionModel().getSelectedItem().equals(libraryTab)) {
-                                    ((LibraryTab) libraryTab).dropEntry(stateManager.getLocalDragboard().getBibEntries());
+                                    LibraryTab libraryTabCasted = (LibraryTab) libraryTab;
```

Comment:
Can you please assign tab and libraryTabCasted more expressive names. For instance, you can rename them to *destinationTabNode* and *destinationLibraryTab*.

Reply:
I can confirm that I will be changing the `libraryTabCasted` to `destinationLibraryTab`- thanks for that - , however, for `tab` I have not added that and as you can see it has been used by the name tab many times in this file. So I do not feel ok changing that? Can you please let me know what you think?

Reply:
Yeah, you can go ahead and rename it. I looked into the other code and destinationLibraryTabNode *(or destinationTabNode)* fits well in there too.